element-connector/src/main/java/org/eclipse/californium/elements/CorrelationContext.java

```diff
@@ -27,6 +28,13 @@
 public interface CorrelationContext {
 
 	/**
+	 * Gets the set of keys used to match a correlation contexts.
+	 * 
+	 * @return set of keys
+	 */
+	Set<String> getMatchingKeys();
```

Comment:
I would named it : `getKeys()` and I wouldn't talk about "matching" in the javadoc.